nixos/modules/services/web-apps/pingvin-share.nix

```
@@ -0,0 +1,233 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.services.pingvin-share;
+  inherit (lib) mkOption mdDoc mkEnableOption mkIf mkPackageOptionMD types;
+in
+
+{
+  options = {
+    services.pingvin-share = {
+      enable = mkEnableOption (mdDoc "Pingvin Share, a self-hosted file sharing platform and an alternative for WeTransfer.");
+
+      user = mkOption {
+        type = types.str;
+        default = "pingvin";
+        example = "root";
+        description = mdDoc ''
+          User account under which Pingvin Share runs.
+        '';
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "pingvin";
+        example = "medias";
+        description = mdDoc ''
+          Group under which Pingvin Share runs.
+        '';
+      };
+
+      openFirewall = mkOption {
+        type = types.bool;
+        default = false;
+        description = mdDoc ''
+          Whether to open the firewall for the port in {option}`services.pingvin-share.frontend.port`.
+        '';
+      };
+
+      settings = {
+        dataDir = mkOption {
+          type = types.path;
+          default = "/var/lib/pingvin-share";
+          example = "/var/wwww/pingvin";
+          description = mdDoc ''
+            The path to the data directory in which Pingvin Share will store its data.
+          '';
+        };
+
+        hostname = mkOption {
+          type = types.nullOr types.str;
+          default = null;
+          example = "pingvin-share.domain.tdl";
+          description = mdDoc ''
+            The domain name of your instance. If null, the redirections will be made to localhost.
+          '';
+        };
+
+        https = mkOption {
+          type = types.bool;
+          default = false;
+          example = true;
+          description = mdDoc ''
+            Whether to enable HTTPS for the domain.
+          '';
+        };
+      };
+
+      backend = {
+        package = mkPackageOptionMD pkgs [ "pingvin-share" "backend" ] { };
+
+        port = mkOption {
+          type = types.port;
+          default = 8080;
+          example = 9000;
+          description = mdDoc ''
+            The port that the backend service of Pingvin Share will listen to.
+          '';
+        };
+      };
+
+      frontend = {
+        package = mkPackageOptionMD pkgs [ "pingvin-share" "frontend" ] { };
+
+        port = mkOption {
+          type = types.port;
+          default = 3000;
+          example = 8000;
+          description = mdDoc ''
+            The port that the frontend service of Pingvin Share will listen to.
+          '';
+        };
+      };
+
+      nginx = {
+        enable = mkEnableOption (mdDoc ''
+          exposing Pingvin Share with a Nginx reverse proxy.
+        '');
```

```suggestion
        enable = mkEnableOption (mdDoc "exposing Pingvin Share with a Nginx reverse proxy");
```

The generated text will look like

	Wether to enable exposing Pingvin Share with a Nginx reverse proxy.

which reads strange